README.md

```diff
@@ -17,6 +17,17 @@ Dogecoin is a community-driven cryptocurrency that was inspired by a Shiba Inu m
 
 **Website:** [dogecoin.com](https://dogecoin.com)
 
+## Dogecoin Policy 📜
+
+Current blockchain consensus of the Dogecoin network. Such fees, such speed !
+
+| Parameter       | Value     |
+| :-------------- | ---------:|
+| Transaction fee |   0.01 Ð  |
+| Minimum spend   |   0.01 Ð  |
```

Comment:
This is must currently still stay 1 DOGE due to #2603, and I think we should put the size here that is recommended as `MIN_CHANGE`, not the dust limit... open to discussion.

```suggestion
| Minimum output  |   1.02 Ð  |
```

Reply:
So you are going to reduce the dust limit, but you want to keep the previous value for the wallet until enough node updates ? People will have the ability to use this smaller spent with all of your PRs linked in #2603, but it's to suggest people to use `MIN_CHANGE` value ?

Reply:
> So you are going to reduce the dust limit, but you want to keep the previous value for the wallet until enough node updates ?

Not me, I just fix bugs and do recommendations based on research/logic/knowledge of how this chain works. The network will lower it... maybe. I'm just giving the network the ability to do it. Shibes in turn will be able to send lower dust amounts if/when 1.14.5 has prominence and the defaults get adopted, in both relay and mining because the affected policy is a mempool policy. Unless there is a path from a wallet to a miner that has 1.14.5 or a manually lowered hard dust limit on 1.14.4, the dust limit is a hard 1.0 DOGE right now and transactions will be discarded. Note that the recommended parametrization to be published is in the wallet code, not in the relay network code - the latter just enables the former (and makes it more flexible by not enforcing the exact same values.)

> People will have the ability to use this smaller spent with all of your PRs linked in #2603,

Only if it's adopted... from a documentation point of view you cannot publish a recommendation before it can be realized - that would be misleading. The PRs you refer to are enabling a future lower recommendation and correcting the recommendation we wanted to do now that the bug we found prohibits (we wanted 0.01 DOGE but we cannot recommend that right now.) I'm sorry if this is all complicated, like I said, I've just been fixing the bug that prevents relay of dust rather than penalizing it. I don't run Dogecoin - that's what everyone does together and things like this take time to be corrected in the field because it's a question of adoption, not dictation - that's what banks and governments do, we're neither. I'd also rather not have these kind of bugs out in the field and if I could turn back time to prevent it, I would - don't think for a moment that I'm enjoying wasting my time on fixing bugs that could have easily been prevented. It's an outrage.

> it's to suggest people to use MIN_CHANGE value ?

The reason why I suggest using the minimum change value instead of the newly introduced discard threshold is because you don't want to send people outputs that are more expensive to spend than they are worth. The minimum change value has a calculation that makes sure that there's enough fee to bump a transaction 10x with RBF or make a child transaction spend itself (and if needed, bump through CPFP). In currently normal scenarios where there is no scarcity of block space, this would mean that the output has enough fee with it to be spent by itself, as sending someone an amount of DOGE at exactly the dust limit means that all they can do with it is discard it, or get more DOGE from another output to spend it with. What if you have no other DOGE?

So if/once the network adapts the bug fix and recommendation from 1.14.5, the minimum output recommendation can be lowered to 0.03 DOGE in a subsequent release, which is 3x minimum fee per kb, and that is close to what Bitcoin does (in the most basic of their coin selection algorithms.) I think that makes more sense as a recommendation than saying "just send the dust limit", because the latter would not take into account that that output still needs to get spent later.

Reply:
Ok fine, misunderstanding of the process for recommendations.